docs/pages/router/basics/layout.mdx

```diff
@@ -0,0 +1,166 @@
+
+```tsx _layout.tsx
+import { Stack } from 'expo-router';
+
+export default function StackLayout() {
+  return <Stack />;
+}
+```
+
+When you navigate to `/products`, it will first go to the default route, which is **products/index.tsx**. If you navigate to `/products/123`, then that page will be pushed onto the stack. By default, the stack will render a back button in the header that will pop the current page off the stack, returning the user to the previous page. Even when a page isn't visible, if it is still pushed onto the stack, it is still being rendered.
+
+The `Stack` component implements React Navigation's native stack, and can use the same screen options. However, you do not have to define the pages specifically inside the navigator. The files inside the folder will be automatically treated as eligable routes in the stack. However, if you want to define screen options, you can add a `Stack.Screen` component inside the `Stack` component. The `name` prop should match the route name, but you do not need to supply a `component` prop; Expo Router will map this automatically:
+
+```tsx _layout.tsx
+import { Stack } from 'expo-router';
+
+export default function StackLayout() {
+  return (
+    <Stack>
+      <Stack.Screen name="[productId]" options={{ headerShown: false }} />
+    </Stack>
+  );
+}
+```
+
+While it is possible to nest navigators, be sure to only do so when it is truly needed. In our example, if you wanted to then push **products/accessories/index.tsx** onto the stack, it's not necessary to have an additional **\_layout.tsx** in the **accessories** folder with a `Stack` navigator. That would define another stack inside the first one. It is fine to add folders that only affect the URL but otherwise use the same navigator as the parent folder.
+
+## Tabs
+
+Much like a stack, you can implement a tab navigator in your layout file, and all the routes directly inside that folder will be treated as tabs. Consider this file structure:
+
+<FileTree
+  files={[
+    ['app/(tabs)/\_layout.tsx'],
+    ['app/(tabs)/index.tsx'],
+    ['app/(tabs)/feed.tsx'],
+    ['app/(tabs)/profile.tsx'],
+  ]}
+/>
+
+In the **\_layout.tsx** file, return a `Tabs` component:
+
+```tsx _layout.tsx
+import { Tabs } from 'expo-router';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+
+export default function TabLayout() {
+  return (
+    <Tabs>
+      <Tabs.Screen
+        name="index"
+        options={{
+          title: 'Home',
+          tabBarIcon: ({ color }) => <MaterialIcons size={28} name="house.fill" color={color} />,
+        }}
+      />
+      <!-- Add more tabs here -->
+    </Tabs>
+  );
+}
+```
+
+This will cause the **index.tsx**, **feed.tsx**, and **profile.tsx** files to appear together in the same bottom tabs navigator. This `Tabs` component uses React Navigation's native bottom tabs and supports the same options.
```

Comment:
```suggestion
This will cause the **index.tsx**, **feed.tsx**, and **profile.tsx** files to appear together in the same bottom tabs navigator. This `Tabs` component uses [React Navigation's native bottom tabs](https://reactnavigation.org/docs/bottom-tab-navigator/) and supports the same options.
```